docs/run/rest-api-primitives.mdx

```diff
@@ -0,0 +1,490 @@
+    }
+job_input = {
+    'program_id': 'estimator',
+    "backend": backend, 
+    "hub": "hub1",
+    "group": "group1",
+    "project": "project1",
+    "start_session": "False", # set to False if you just need to run a single job.  
+    "params": {
+        "pubs": [ #primitive unified blocs (PUBs) containing one circuit each. c.f. https://docs.quantum.ibm.com/api/migration-guides/v2-primitives
+            [resulting_qasm, # QASM circuit
+             {"IIZII": 1, "XIZZZ": 2.3}, # Observable
+             None # parameter values
+             ]], 
+        "version": 2 # this defines the version of the Qiskit Runtime Primitive to use, c.f. https://docs.quantum.ibm.com/api/migration-guides/v2-primitives
+}}
+
+response = requests.post(url, headers=headers, json=job_input)
+
+if response.status_code == 200:
+    job_id = response.json().get('id')
+    print("Job created:",response.text)
+else:
+    print(f"Error: {response.status_code}")
+```
+
+
+### 4. Check Job Status and Get Results
+
+Next, pass the `job_id` to the API:
+
+```python
+response_status_singlejob= requests.get(url+'/'+job_id, headers=headers)
+response_status_singlejob.json().get('state')
+```
+Output
+
+```text
+>>> Job ID: 58223448-5100-4dec-a47a-942fb30edcad
+>>> Job Status: JobStatus.RUNNING
+```
+
+Get job results:
+```python
+response_result= requests.get(url+'/'+job_id+'/results', headers=headers)
+
+res_dict=response_result.json()
+
+estimator_result=res_dict['results']
+print(estimator_result)
+```
+Output
+```text
+[{'data': {'evs': 0.7428980350102542, 'stds': 0.029884014518789213, 'ensemble_standard_error': 0.03261147170624149}, 'metadata': {'shots': 10016, 'target_precision': 0.01, 'circuit_metadata': {}, 'resilience': {}, 'num_randomizations': 32}}]
+```
+
+### 5. Working with Runtime Options
+
+Error mitigation techniques allow users to mitigate circuit errors by modeling the device noise at the time of execution. This typically results in quantum pre-processing overhead related to model training and classical post-processing overhead to mitigate errors in the raw results by using the generated model.
+
+The error mitigation techniques built in to primitives are advanced resilience options. To specify these options, use the resilience_level option when submitting your job.
+
+You will see examples of three such error mitigation / suppression techniques when used with the Estimator Primitive. The examples given below demonstrate the default options for error mitigation / suppression techniques but more options for [Dynamical decoupling](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/options/dynamical_decoupling_options.py), [Twirling](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/options/twirling_options.py) and [TREX + ZNE](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/options/resilience_options.py) can be further explored. 
```

Comment:
Instead of all these links I think it would be good to link to the new error mitigation techniques page that we just released in docs: https://docs.quantum.ibm.com/run/error-mitigation-explanation

Reply:
Wow, this is quite helpful. I hadn't come across this page before.